__tests__/writePass.test.js

```diff
@@ -363,13 +363,24 @@ async function databaseOperation(version, mode) {
                 max: 25
               }
             },
-            status: {stateReason: '', state: 'open'},
+            bookingDays: {
+              "1": true,
```

Comment:
CSC: We start at 0 lol

Reply:
Yeah I know - `luxon` weekdays are 1-indexed starting on Monday though. Can make it 0-indexed on Sunday as per the ticket but then we gotta do some conversion which feels... snowflakey